lms/djangoapps/teams/serializers.py

```diff
@@ -1,13 +1,15 @@
 """Defines serializers used by the Team API."""
-
+from copy import deepcopy
 from django.contrib.auth.models import User
 from django.db.models import Count
+from django.conf import settings
 
 from rest_framework import serializers
 
 from openedx.core.lib.api.serializers import CollapsedReferenceSerializer, PaginationSerializer
 from openedx.core.lib.api.fields import ExpandableField
 from openedx.core.djangoapps.user_api.serializers import UserSerializer
+from openedx.core.djangoapps.user_api.accounts.serializers import AccountFullUserProfileReadOnlySerializer
```

Comment:
Another thought: it isn't always full, depending upon the configuration specified.